account/account.go

```diff
@@ -32,7 +32,7 @@ import (
 // FetchAssertionFromStore retrieves an assertion from the store
 var FetchAssertionFromStore = func(modelType *asserts.AssertionType, headers []string) (asserts.Assertion, error) {
 	var user *auth.UserState
-	var authContext auth.AuthContext
+	var authContext store.DeviceAndAuthContext
```

Comment:
version of snapd was updated and this package was moved

Reply:
I don't love that this PR includes a stealth dependency update - was the snapd dependency not pinned previously?

Reply:
Yes, it was, now the version was generated by `go mod init`. Let me try to pin the version to the previous one

Reply:
Hmm, can't pin the version of snapd because of this:
https://bugs.launchpad.net/snapd/+bug/1821959

Reply:
Let's please not introduce unpinned dependencies - that is the path to sadness and woe and surprise breakage and lack of reproducibility